movement/io/load_poses.py

```diff
@@ -190,6 +225,41 @@ def from_dlc_file(
     >>> ds = load_poses.from_dlc_file("path/to/file.h5", fps=30)
     """
 
+    return _from_lp_or_dlc_file(
+        file_path=file_path, source_software="DeepLabCut", fps=fps
+    )
+
+
+def _from_lp_or_dlc_file(
+    file_path: Union[Path, str],
+    source_software: Literal["LightningPose", "DeepLabCut"],
+    fps: Optional[float] = None,
+) -> xr.Dataset:
+    """Loads pose tracking data from a DeepLabCut (DLC) output file or
+    a LightningPose file into an xarray Dataset.
+
+    Parameters
+    ----------
+    file_path : pathlib.Path or str
+        Path to the file containing the DLC predicted poses, either in ".h5"
+        or ".csv" format.
+    fps : float, optional
+        The number of frames per second in the video. If None (default),
+        the `time` coordinates will be in frame numbers.
+    source_software : Can be either "DeepLabCut" or "LightningPose".
+
+    Returns
+    -------
+    xarray.Dataset
+        Dataset containing the pose tracks, confidence scores, and metadata.
+
+    See Also
+    --------
+    movement.io.load_poses.from_dlc_df : Load pose tracks from a DataFrame.
+    from_lp_file
+
+    """
+
     file = ValidFile(
```

Comment:
The expected suffix is `[".csv", ".h5"]` for DLC, but should be only `[".csv"]` for LP. So before calling `ValidFile` you could add sth like:

```python
expected_suffix = [".csv"]
if source_software == "DeepLabCut":
    expected_suffix.append(".h5")
```

and then pass that `expected_suffix` to `ValidFile`